src/test/java/org/openmrs/contrib/qaframework/automation/AdvancedInstallSteps.java

```diff
@@ -1,13 +1,14 @@
 package org.openmrs.contrib.qaframework.automation;
 
+import org.openmrs.contrib.qaframework.RunTest;
```

Comment:
@sherrif10 There's a lot going on in this PR. Could we separate out the re-ordering or imports from the rest of this

Reply:
Thanks @ibacher , Let me address this comment.